spark-indexer/src/main/java/org/uniprot/store/spark/indexer/uniref/converter/DatasetUniRefEntryLightConverter.java

```diff
@@ -111,9 +111,11 @@ private void convertCommonProperties(Row rowValue, UniRefEntryLightBuilder build
 
     private void addMemberInfo(UniRefEntryLightBuilder builder, UniRefMember member) {
         // member accessions
+        int memberType = member.getMemberIdType().getDisplayOrder();
         member.getUniProtAccessions().stream()
```

Comment:
the variable name "memberType"  should be related to order.

Reply:
I think if we put order in the name will make it more confuse... what name do you suggest?